crates/iceberg/src/transaction/append.rs

```diff
@@ -379,6 +364,7 @@ mod tests {
 
         // Attempt to add the existing Parquet files with fast append.
         let new_tx = fast_append_action
+            .with_check_duplicate(false)
```

Comment:
I'm not sure what was initial purpose of the test.
But it looks like to check ability of adding existing parquet files to the table (physically existing not in table).
In which case adding `with_check_duplicate(false)` here is fine.

Reply:
Removed.